utils/build/docker/python/flask-poc.Dockerfile

```diff
@@ -19,6 +19,8 @@ ENV _DD_APPSEC_DEDUPLICATION_ENABLED=false
 ENV DD_BOTOCORE_PROPAGATION_ENABLED=true
 ENV DD_KAFKA_PROPAGATION_ENABLED=true
 
+ENV LOG_LEVEL='DEBUG'
```

Comment:
This was causing lot of flakiness on the python tracer, we removed this line a while ago.